packages/synapse-react-client/package.json

```diff
@@ -60,6 +60,7 @@
     "@tanstack/react-table": "^8.21.3",
     "@tanstack/react-virtual": "^3.13.8",
     "@upsetjs/react": "^1.11.0",
+    "@x0k/json-schema-merge": "^1.0.2",
```

Comment:
Small library (also used by react-jsonschema-form) used to merge two schemas (in this case, the grid validation schema, and the base Entity schema when we have a file view)